ensime_shared/ensime.py

```diff
@@ -279,11 +336,15 @@ def au_buf_leave(self, client, filename):
 
     @execute_with_client()
     def au_cursor_hold(self, client, filename):
-        client.on_cursor_hold(filename)
+        """Handler for CursorHold autocommand event."""
+        self.watchdog_client_server_connection(client)
+        client.unqueue_and_display(filename)
+        client.editor.cursorhold()
```

Comment:
This was moved as-is from `EnsimeClient.on_cursor_hold` so it's not a behavior change in this PR. I don't exactly understand the background of how this achieves "faking a timer" as the link referenced at `Editor.cursohold` describes, but if there's anything wrong with that I'd propose it's brought up in a new issue.